pom.xml

```diff
@@ -3351,7 +3351,7 @@
                 <plugin>
                     <groupId>org.codehaus.mojo</groupId>
                     <artifactId>versions-maven-plugin</artifactId>
-                    <version>2.7</version>
+                    <version>2.11.0</version>
```

Comment:
Commit says 2.10.0 but here it's 2.11.0... Maybe change the commit description to match the right version :)